timescaledb/tutorials/grafana/visualizations/candlestick.md

```diff
@@ -0,0 +1,167 @@
+to get familiar with creating your first dashboard and visualizations. Also
+see [this tutorial on adding variables to Grafana][variables-tutorial].
+
+The examples in this section use these variables and Grafana functions:
+* `$symbol`: a variable used to filter results by stock symbols.
+* `$__timeFrom()::timestamptz` & `$__timeTo()::timestamptz`:
+  Grafana variables. You change the values of these variables by
+  using the dashboard's date chooser when viewing your graph.
+* `$bucket_interval`: the interval size to pass to the `time_bucket`
+  function when aggregating data.
+
+## Create a candlestick with raw data
+Create a candlestick visualization using the raw data in the table `stocks_real_time`.
+
+<procedure>
+
+### Create a candlestick with raw data
+
+  1.  In the query editor, use this SQL to query a Candlestick dataset using specified bucket interval:
+      ```sql
+      SELECT
+      time_bucket($bucket_interval, time) AS time,
+      symbol,
+      FIRST(price, time) AS "open",
+      MAX(price) AS high,
+      MIN(price) AS low,
+      LAST(price, time) AS "close",
+      FROM stocks_real_time
+      WHERE symbol =  $symbol
+      AND time > $__timeFrom()::timestamptz and time < $__timeTo()::timestamptz
+      GROUP BY symbol;
+      ```
+  1.  Click outside of the query editor/the refresh icon to 
+      update the Grafana chart.
+
+  1.  Select the candlestick as your visualization type
+      
+      <img class="main-content__illustration" src="https://s3.amazonaws.com/assets.timescale.com/docs/images/tutorials/visualizations/candlestick/candlestick_visualization.png" alt="Screenshot of the Grafana dashboard. The 'Visualizations' tab is focused. Underneath, 'Candlestick' shows as a visualization type."/>
+
+  1.  Grafana turns the query into a candlestick chart that 
+      looks like this:
+
+      <img class="main-content__illustration" src="https://assets.timescale.com/docs/images/tutorials/visualizations/candlestick/1_min.png" alt="Screenshot of a candlestick chart in Grafana, showing the price distribution of $AMZN."/>
+
+      In this first example, we set the $bucket_interval to 1-min, you can see the price of AMZN ranges between $2120 and $2200. This chart uses hyperfunctions to query the `stock_real_time` table, with a bucket interval of 1 minute. 
+
+</procedure>
+
+Retrieving this data took about 7+ seconds, over two weeks of data which is probably slower than most users would expect when analyzing data. This is where continuous aggregates are particularly useful for data-intensive, time-series applications. 
+
+<img class="main-content__illustration" src="https://s3.amazonaws.com/assets.timescale.com/docs/images/tutorials/visualizations/candlestick/raw_data_exec_time.png" alt="Screenshot of the Grafana query response."/>
+
+<procedure>
+
+  With the use of the $bucket_interval variable, we are able to switch intervals. For example, switching to a 15-minute bucket interval gives you this data. 
```

Comment:
Rather than breaking the same procedure into 2 pieces, I think it's best to move this aside after the entire procedure, in a paragraph that says "hey, you just did it this way, but caggs can make it faster! check them out!"

If you really want this information here, I'd just add it as a numbered step and join the entire thing into 1 procedure